### PR TITLE
Project page updates

### PR DESCRIPTION
**Pull request by blai0264**
# [Create Explore other projects section](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215242)

Description of the task you completed and/or features added. If it is a visual component or feature,
add a screenshot(s) of it.

## Test Instructions

1. yarn dev
2. go to benefits navigator page
3. examine explore other projects section (check responsiveness)
4. check out component code to see if there is any changes needed
5. yarn storybook
6. check organisms/exploreProjects entry

### CODE REVIEW

**Review by will0684 — CHANGES_REQUESTED** [review]
Nice work, coming along nicely. Just a couple things

File: components/molecules/Card.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const Card = (props) => {
         data-cy={props.dataCy}
       >
         {props.showImage ? (
-          <div className="sm:h-80 flex justify-center">
+          <div className="h:[208px] flex justify-center">
```
[the remark this answers is not in the document]
Comment (blai0264) [reply components/molecules/Card.js:31]: good catch

File: components/molecules/Card.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const Card = (props) => {
           ""
         )}
         <div className="flex">
-          <p className="block font-display text-lg text-custom-blue-projects-link font-bold underline underline-offset-[6px] my-1 py-2 px-6 items-center group-hover:text-custom-blue-projects-link-hover">
+          <p className="block font-display text-lg lg:px-2 lg:text-md text-custom-blue-projects-link font-bold underline underline-offset-[6px] m-1 py-2 px-6 items-center group-hover:text-custom-blue-projects-link-hover">
```
[the remark this answers is not in the document]
Comment (blai0264) [reply components/molecules/Card.js:47]: gotcha will do

File: pages/api/shuffle.js
Comment (will0684) [inline pages/api/shuffle.js:1]: This is not an API, it is a utility. [An API is something we want other remote/networked applications/devices to access. ](https://aws.amazon.com/what-is/api/). `/lib/utils` is where these things go.

File: pages/projects/benefits-navigator/index.js [hunk 3]
```diff
@@ -659,6 +680,7 @@ export const getStaticProps = async ({ locale }) => {
       pageData: pageData.sclabsPageV1ByPath,
       updatesData: updatesData.sclabsPageV1List.items,
       dictionary: dictionary.dictionaryV1List,
+      allProjects: await shuffle(allProjects.sclabsPageV1List.items),
```
Comment (will0684) [inline pages/projects/benefits-navigator/index.js:683]: `await` is not necessary here

File: pages/api/shuffle.js
Comment (blai0264) [reply pages/api/shuffle.js:1]: ok noted 👍🏾

**Review by will0684 — APPROVED** [review]
Great work!

**Pull request merged**